### DISCUSSION TITLE
Seeking Community Feedback and Testing for New Organization Module

### DISCUSSION THREAD

**Discussion by sinamics** [sinamics/ztnet#221] [General]
Hi

I would like to get some feeback on the functionality and layout. 
At this stage, everything related to the organizations is in English, as translations are the only component yet to be integrated.

Organization PR: #99
Other related discussions: #169

What I am looking for:

- Usability Feedback: How intuitive and user-friendly do you find the interface? Are there any aspects of the user experience that could be improved?
- Functionality Testing: Please try out the various features and report any bugs or issues you encounter. I am particularly interested in any unexpected behavior or glitches.
- Design and Layout Suggestions: Any thoughts on the aesthetic aspects or the layout of the application would be highly valuable. I am open to suggestions for making the interface more engaging and easier to navigate.

**!NOTE:** take backup and do not test in production!
For Organization testing use this image:

```yml
  ztnet:
    image: sinamics/ztnet:organization-49676cf
````

or, if you are using standalone version, you can install the organization branch using the bellow command:
```bash
curl http://install.ztnet.network | sudo bash -s -- -b organization
```

**Comment by sinamics** [comment]
New version available with organization included. 0.5.0

**Comment by tinola** [comment]
I will test soon I hope.
Unfortunately after upgrade to FreeBSD 14-RELEASE the prisma client is not building. I've created [PR](https://github.com/prisma/prisma/issues/22228) and waiting for fix...
